lib/coinbase/wallet.rb

```diff
@@ -132,14 +132,15 @@ def fetch_wallets_page(page)
     #   If nil, a new seed will be generated. If the empty string, no seed is generated, and the Wallet will be
     #   instantiated without a seed and its corresponding private keys.
     #   with the Wallet. If not provided, the Wallet will derive the first default address.
-    def initialize(model, seed: nil)
+    def initialize(model, seed: nil, network_id: nil)
```

Comment:
The network_id is also present on the model so we do not need to pass this in:

https://github.com/coinbase/coinbase-sdk-ruby/blob/6b92ca0cb3c9c590e816196e96fc56c10a907dff/lib/coinbase/wallet.rb#L291-L295

We definitely should serialize it on the way out and we can do a sanity check on the way in too.

This exposes a pretty important concept/flaw that unfortunately we scoped wallets to network.,